airflow/www/app.py

```diff
@@ -980,25 +984,27 @@ def action(self):
                     t.task_id
                     for t in task.get_flat_relatives(upstream=True)]
             TI = models.TaskInstance
+            dates = utils.date_range(start_date, end_date)
             tis = session.query(TI).filter(
                 TI.dag_id == dag_id,
-                TI.execution_date == execution_date,
+                TI.execution_date.in_(dates),
```

Comment:
Tried this out, but doesn't really work since you need to know which ones are successes so you don't try to create new records later on. So will keep it this way.